cli/tests/e2e/snapshots/test_baseline/test_staged_changes/error.txt

```diff
@@ -0,0 +1,29 @@
+               
+               
+┌─────────────┐
+│ Scan Status │
+└─────────────┘
+  Scanning 1 file tracked by git with 1 Code rule:
+  Scanning 1 file.
+  Current version has 2 findings.
+
+Switching repository to baseline commit '4b63ca493df05e223404a80ac5f30d2aa830a704'.
+  Will report findings introduced by these commits (may be incomplete for shallow checkouts):
```

Comment:
Maybe just link to docs and include warnings like "(may be incomplete for shallow checkouts)" in the docs? Alternatively, if this is a pretty common scenario, let's improve the warning to make it clear what condition they're looking for and what they can do. Something like:

`Note: You may see findings that weren't introduced by you. This can happen when we perform a shallow checkout. We do this because ___. You can fix that by ___`